pkg/quarkus/v1alpha/init_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/pflag"
+
+	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+)
+
+var _ = Describe("v1", func() {
+	successInitSubcommand := &initSubcommand{domain: "testDomain"}
+	failureInitSubcommand := &initSubcommand{
+		domain:      "testDomain",
+		projectName: "?&fail&?",
+		commandName: "failureTest",
+	}
```

Comment:
Put in a `BeforeEach` block. that way each test has a fresh version that hasn't been affected by any changes from subsequent test.